webapp/src/js/controllers/reports.js

```diff
@@ -255,8 +255,10 @@ angular
     };
 
     $scope.search = function() {
-      $state.go('reports.detail', { id: null }, { notify: false });
-      clearSelection();
+      if($scope.filters.search) {
+        $state.go('reports.detail', { id: null }, { notify: false });
```

Comment:
Why do we need this line here but not in contacts?

Reply:
Have updated contacts with this too. It was not necessary to clear the selected contact but it does update the url to be just .../contacts instead of the selected contact url.